content/blog/2022/04/camel-karavan-0.0.14/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Karavan Designer Preview Release 0.0.14"
+date: 2022-04-18
+authors: [mgubaidullin]
+categories: ["Releases","Tooling"]
+preview: "What's new in Karavan 0.0.14"
+---
+
+![logo](karavan-logo-light.png)
+
+What's going on with Apache Camel Karavan? 
+
+Karavan is progressing! The following cool staff was done since the last [post](/blog/2022/02/camel-karavan-0.0.12):
+
+# REST API
+
+With the aid of the new clean REST DSL in Camel 3.16  Karavan can help to design REST API.
+Starting from version 0.0.13 Karavan requires Camel 3.16+ runtime
+
+![rest](./rest.png)
+
+# User Interface improvements
+
+## New theme
+
+Karavan comes with the new theme aligned to VSCode build-in theme.
+
+![theme](./theme-light.png)
+
+![theme](./theme-dark.png)
+
+## New VSCode view
```

Comment:
The "official" spelling is with a space: VS Code
![Screenshot from 2022-04-15 11-12-53](https://user-images.githubusercontent.com/1105127/163552032-fe5bd927-d307-41aa-b670-31d9791a3457.png)

Reply:
Thanks @apupier